### PR TITLE
Added RXG, TQ1, and RXR Segment Definitions.

### PR DESCRIPTION
**Pull request by dave92082**
### Overview ###
Add segments to support IHE Standards for Patient Care Devices Domain.

### New Segments Added ###
* RXG (HL7 v2.6)
    * Pharmacy/Treatment Give Segment
* TQ1 (HL7 v2.6)
    * Timing/Quantity Segment
* RXR (HL7 v2.6)
    * Pharmacy/Treatment Route Segment

### Updated Segments ###
* OBX
* PID
* ORC

### CODE REVIEW

**Comment by evanlucas** [comment]
Wow. Thanks for the contribution! Can I ask you to please remove the comments from the segment files?

Can you also make sure the indentation is the same in all of those files?

Also, with the three segments being updated, this will have to be a semver-major change, but that is fine with me.

Thanks!!!

**Comment by dave92082** [comment]
Thought the comments would be helpful for identifying data types. I'll clean them all up and update it shortly.

File: lib/segments/obx.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ exports.fields = [
   , 'SetID'
   , 'ValueType'
   , 'ObservationIdentifier'
-  , 'ObservationSubID'
+  , 'ObservationSub-ID'
```
Comment (evanlucas) [inline lib/segments/obx.js]: Hmmm I would think we should remove the hyphen here

Comment (dave92082) [inline lib/segments/obx.js]: Yup, completely agree! I thought I got them all.

**Comment by evanlucas** [comment]
Thanks again for the contribution!!

**Comment by evanlucas** [comment]
published as v4.0.0

**Pull request merged**